nav2_astar_planner/src/astar_planner.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include <string>
```

Comment:
I know this is outside the scope of this PR, but reading this made me think of it. Since we have a dijkstra planner now should we remove this astar_planner stub? Or should we rename it nav2_example_planner?

Reply:
Yes, I think we should rename it to be an example planner. It can either be a fully functioning but very simple planner, or just a shell/stub.